app/jobs/day_check_job.rb

```diff
@@ -0,0 +1,17 @@
+class DayCheckJob < ApplicationJob
+  queue_as :default
+
+  def perform(*args)
+    days = get_unchecked_days
+    return if days.empty?
+    days.each do |day|
+      DayCheckMailer.with(day_habit: day).day_check_email.deliver_later!
+    end
+  end
+
+  private
+
+  def get_unchecked_days
+    days = Day.where(status: false, date: Date.today)
```

Comment:
it would be ok if you just add the query without using a variable: 
```suggestion
    Day.where(status: false, date: Date.today)
```

Reply:
Great observation, thank you a lot mate!